app/graphql/concerns/custom_payload_type.rb

```diff
@@ -0,0 +1,37 @@
+module CustomPayloadType
+  extend ActiveSupport::Concern
+
+  CRUD_OPERATION = %w[create update delete].freeze
+
+  class_methods do
+    # NOTE: Hook Override - https://github.com/rmosolgo/graphql-ruby/issues/2737
+    def generate_payload_type
+      resolver_name = generate_resolver_name
+      resolver_fields = fields
+
+      Class.new(object_class) do
+        graphql_name("#{resolver_name}Payload")
+        description("Autogenerated return type of #{resolver_name}")
+
+        if resolver_fields.keys.exclude?('errors')
+          field :errors, [Errors::UserError],
+            null: true,
+            description: 'Graphql Errors'
```

Comment:
We can't be sure it'll only return a `UserError`.  It's entirely possible that they'll receive a server error or something else.

This should almost certainly return `[Errors::ErrorInterface]` instead.

Reply:
I didn't realize we can straight up return an `ErrorInterface` 😆.